.github/workflows/helm-releaser-all-charts.yaml

```diff
@@ -96,11 +96,9 @@ jobs:
               echo "Checking latest GH pages build status --> $status"
               sleep 5
 
-              # checking if GH pages built doesn't take too much time or it didn't go into erroring state.
-              # waiting time for build to succeed is set to 10 minutes.
               counter=$(expr $counter + 1)
-              if [[ $status =~ error.* ]] || [[ $counter == 120 ]]; then
-                echo Something went wrong. Please check GH's pages issues.
+              if [[ $status == \"errored\" ]] || [[ $counter == 120 ]]; then
```

Comment:
typo? or there is the string with "errored"?

Reply:
Yeap. it says not an error but errored.

Reply:
You can check here: https://github.com/kubeshop/helm-charts/runs/4186834233?check_suite_focus=true